### PR TITLE
rtx: update to 1.35.8

### PR DESCRIPTION
**Pull request by TeddyDD**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/rtx/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'rtx'
 pkgname=rtx
-version=1.34.1
+version=1.35.0
```
Comment (classabbyamp) [inline srcpkgs/rtx/template:3]: 1.35.1 avail

Comment (TeddyDD) [reply srcpkgs/rtx/template]: bumped

Comment (classabbyamp) [reply srcpkgs/rtx/template]: seems it's up to .7 now :|

Comment (TeddyDD) [reply srcpkgs/rtx/template]: bumped

**Pull request merged**